components/brave_rewards/resources/extension/brave_rewards/components/app.tsx

```diff
@@ -153,14 +178,31 @@ export class RewardsPanel extends React.Component<Props, State> {
     this.actions.createWallet()
   }
 
+  getClaimedGrants = (grants?: RewardsExtension.Grant[]) => {
```

Comment:
I see that this function is the same as in panel.tsx. Can you please extract this to util function, so that it's not duplicated

Reply:
@NejcZdovc added to `utils`